config/chassis_modules.py

```diff
@@ -30,8 +30,10 @@ def shutdown_chassis_module(db, chassis_module_name):
 
     if not chassis_module_name.startswith("SUPERVISOR") and \
        not chassis_module_name.startswith("LINE-CARD") and \
-       not chassis_module_name.startswith("FABRIC-CARD"):
-        ctx.fail("'module_name' has to begin with 'SUPERVISOR', 'LINE-CARD' or 'FABRIC-CARD'")
+       not chassis_module_name.startswith("FABRIC-CARD") and \
+       not chassis_module_name.startswith("DPU") and \
+       not chassis_module_name.startswith("SWITCH"):
```

Comment:
Do we need switch here?

Reply:
This is a Microsoft ask. One use case I can see is  that "reboot SWITCH".

Reply:
Fixed